docs/content/installation/integrations/app-protect-waf/compile-waf-policies.md

```diff
@@ -0,0 +1,359 @@
+- A working [NGINX Management Suite](https://docs.nginx.com/nginx-management-suite/installation/) instance.
+- An [NGINX Management Suite user](https://docs.nginx.com/nginx-management-suite/admin-guides/rbac/rbac-getting-started/) for API requests.
+- A NGINX Ingress Controller [deployment with NGINX App Protect WAF]({{< relref "/installation/integrations/app-protect-waf/installation.md" >}}).
+
+## Create a new security policy
+
+{{< tip >}} You can skip this step if you intend to use an existing security policy. {{< /tip >}}
+
+First, create a [new security policy](https://docs.nginx.com/nginx-management-suite/nim/how-to/app-protect/manage-waf-security-policies/#create-security-policy) using the API: this will require the use of a tool such as [`curl`](https://curl.se/) or [Postman](https://www.postman.com/)
+
+You will use the API to upload JSON files for the policy, which will be the same method for creating the bundle later.
+
+Create the file `simple-policy.json` with the contents below:
+
+```json
+{
+  "metadata": {
+    "name": "Nginxbundletest",
+    "displayName": "Nginxbundletest",
+    "description": "Ignore cross-site scripting is a security policy that intentionally ignores cross site scripting."
+  },
+  "content": "ewoJInBvbGljeSI6IHsKCQkibmFtZSI6ICJzaW1wbGUtYmxvY2tpbmctcG9saWN5IiwKCQkic2lnbmF0dXJlcyI6IFsKCQkJewoJCQkJInNpZ25hdHVyZUlkIjogMjAwMDAxODM0LAoJCQkJImVuYWJsZWQiOiBmYWxzZQoJCQl9CgkJXSwKCQkidGVtcGxhdGUiOiB7CgkJCSJuYW1lIjogIlBPTElDWV9URU1QTEFURV9OR0lOWF9CQVNFIgoJCX0sCgkJImFwcGxpY2F0aW9uTGFuZ3VhZ2UiOiAidXRmLTgiLAoJCSJlbmZvcmNlbWVudE1vZGUiOiAiYmxvY2tpbmciCgl9Cn0="
+}
+```
+
+{{< warning >}}
+
+The `content` value must be base64 encoded or you will encounter an error.
+
+{{< /warning >}}
+
+In the same directory you created `simple-policy.json`, create a POST request for NGINX Instance Manager using the API.
+
+```shell
+curl -X POST https://{{NMS_FQDN}}/api/platform/v1/security/policies \
+    -H "Authorization: Bearer <access token>" \
+    -d @simple-policy.json
+```
+
+You should receive an API response similar to the following output, indicating the policy has been successfully created.
+
+```json
+{
+    "metadata": {
+        "created": "2024-06-12T20:28:08.152171922Z",
+        "description": "Ignore cross-site scripting is a security policy that intentionally ignores cross site scripting.",
+        "displayName": "Nginxbundletest",
+        "externalId": "",
+        "externalIdType": "",
+        "modified": "2024-06-12T20:28:08.152171922Z",
+        "name": "Nginxbundletest",
+        "revisionTimestamp": "2024-06-12T20:28:08.152171922Z",
+        "uid": "6af9f261-658b-4be1-b07a-cebd83e917a1"
+    },
+    "selfLink": {
+        "rel": "/api/platform/v1/security/policies/6af9f261-658b-4be1-b07a-cebd83e917a1"
+    }
+}
+```
+
+{{< important >}}
+
+Take note of the *uid* field. 
+
+It is one of two unique IDs we will use to download the bundle: it will be referenced as *policy-UID*.
+
+{{< /important >}}
```

Comment:
```suggestion
{{< important >}}

Take note of the *uid* field: `"uid": "6af9f261-658b-4be1-b07a-cebd83e917a1"`
It is one of two unique IDs we will use to download the bundle: it will be referenced as *policy-UID*.

{{< /important >}}
```